### DISCUSSION TITLE
Thanks!

### DISCUSSION THREAD

**Discussion by fbarthez** [ashish10alex/vscode-dataform-tools#37] [General]
Hey @ashish10alex, just wanted to thank you for building this! Definitely willing to buy you a coffee. :)

**Comment by ashish10alex** [comment]
@fbarthez , thank you for the kind words. Not sure if you were serious about the coffee thing but here [the link](https://buymeacoffee.com/ashishalexj) to buy me coffee :)

**Reply by ashish10alex** [reply]
wow!, you were serious haha, thank you so much for the coffees :)
